### PR TITLE
Update explainer to include feature detection and new syntax

### LINKED ISSUE #170: Update example to show feature detection
Now that we're back to using `navigator.credentials`, our simple feature detection mechanism of `if ('identity' in navigator)` won't work. Presumably we'd recommend `if ('DigitalCredential' in window)` instead, right? Perhaps we should add this to the example in the explainer to make it clear? 

I know of one developer who wasn't sure how best to feature-detect our origin trial and so instead looked for a Chrome Android UA string with Chrome version >= 128. Obviously that's not what we want people doing since the API will become available on other browsers too. Our examples should document our best-practice recommendations.

### PR DESCRIPTION
**Pull request by timcappalli**
Closes #170

### CODE REVIEW

**Review by OR13 — APPROVED** [review]
I wonder if adding fetch would improve this example.

**Comment by timcappalli** [comment]
> I wonder if adding fetch would improve this example.

the full example that includes fetch is linked below the code snippet.

File: explainer.md [hunk 1]
```diff
@@ -37,33 +37,58 @@ Here is an example of how the  the API might be used in practice:
...
-      requests: [oid4pv],
-    },
-  });
-  // To be decrypted on the server...
-  const encryptedData = digitalCredential.data;
-}
+async function requestCredential() {
+
+  // Check for Digital Credentials API support
+  if (typeof window.DigitalCredential !== 'undefined') {
```
Comment (samuelgoto) [inline explainer.md:48]: Maybe return early here? e.g.

```suggestion
  if (typeof window.DigitalCredential !== 'undefined') {
    // API not supported, return undefined.
    return;
  }
```

**Review by samuelgoto — APPROVED** [review]
LGTM

**Pull request merged**